api/src/modules/custom-projects/conservation.project.ts

```diff
@@ -4,15 +4,15 @@ import {
   DEFAULT_STUFF,
 } from '@api/modules/custom-projects/project-config.interface';
 import { BaseDataView } from '@shared/entities/base-data.view';
-import { CostInputs } from '@api/modules/custom-projects/cost-inputs.interface';
+import { CostInputsDeprecated } from '@api/modules/custom-projects/cost-inputs.interface';
 import { ModelAssumptions } from '@shared/entities/model-assumptions.entity';
 
 export class ConservationProject {
   name: string;
   activity: ACTIVITY.CONSERVATION;
   ecosystem: string;
   countryCode: string;
-  costInputs: CostInputs;
+  costInputs: CostInputsDeprecated;
```

Comment:
Will remove this Deprecated stuff once we implement the calculations. Or should I remove it? We would also need to remove all old calculation classes probably, they might be useful for reference